LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Adel Ghabboun
```

Comment:
I made a similar comment on the snapshot policy PR if this should say Microsoft or not for any legal reasons. Maybe @paulomarquesc knows more about this?